src/polyfill/previewer.js

```diff
@@ -99,11 +99,22 @@ class Previewer {
 		return template.content;
 	}
 
-	removeStyles(doc=document) {
+	removeStyles(doc=document, content=null) {
 		// Get all stylesheets
 		const stylesheets = Array.from(doc.querySelectorAll("link[rel='stylesheet']:not([data-pagedjs-ignore], [media~='screen'])"));
 		// Get inline styles
 		const inlineStyles = Array.from(doc.querySelectorAll("style:not([data-pagedjs-inserted-styles], [data-pagedjs-ignore], [media~='screen'])"));
+
+		// Get all styles from the content (content of the template cant be queried by document.querySelector directly)
```

Comment:
document.querySelector won't be able to gather styles within a <template> element, those need a separate query which will target their content (e.g. template.content.querySelector).